presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -3007,13 +3007,23 @@ public void testBucketSortedTables()
             throws Exception
     {
         SchemaTableName table = temporaryTable("create_sorted");
-        SchemaTableName tableWithTempPath = temporaryTable("create_sorted_with_temp_path");
         try {
             doTestBucketSortedTables(table, false, ORC);
-            doTestBucketSortedTables(tableWithTempPath, true, ORC);
         }
         finally {
             dropTable(table);
+        }
+    }
+
+    @Test
+    public void testBucketSortedTablesTempPath()
```

Comment:
this test refactoring looks unrelated. pull it out changes in this file to a separate commit